scripts/Source/sslConfigMenu.psc

```diff
@@ -2928,13 +2977,13 @@ Form[] function GetItems(Actor ActorRef, bool FullInventory = false)
 	endIf
 endFunction
 
-string function GetItemName(Form ItemRef)
+string function GetItemName(Form ItemRef, string AltName = "$SSL_Unknown")
```

Comment:
just because in some cases the $SSL_Unknown is not translated allowing me to use some alternative like the "?"